crates/pop-parachains/src/errors.rs

```diff
@@ -36,6 +36,9 @@ pub enum Error {
 	/// The dispatchable function is not supported.
 	#[error("The dispatchable function is not supported")]
 	FunctionNotSupported,
+	/// Failed to retrieve the image tag.
+	#[error("Failed to retrieve image tag.")]
+	ImageTagRetrievalFailed,
```

Comment:
Slowly ecoming a kitchen sink of mismatched errors. Please consider module specific errors in future efforts, which can then be bubbled up to the main error.